doc/running-cljdoc-locally.md

```diff
@@ -10,6 +10,15 @@ to instead [run cljdoc locally from docker](#run-cljdoc-locally-from-docker).
 
 ## Run cljdoc Locally from Source
 
+### Dependencies cljdoc
+
+To run cljdoc from Source, you must install these dependencies:
+ - nodejs
+ - clojure and its command line tools
+
+To install nodejs, follow [instructions here](https://nodejs.org/en/).
+To install clojure and tools, follow [instructions here](https://clojure.org/guides/getting_started).
+
```

Comment:
To keep this dense maybe you can combine the list of dependencies with the installation instructions? E.g.

- nodejs ([installation instructions](https://nodejs.org/en/))
- Clojure CLI tools ([installation instructions](https://clojure.org/guides/getting_started))

```md
- nodejs ([installation instructions](https://nodejs.org/en/))
- Clojure CLI tools ([installation instructions](https://clojure.org/guides/getting_started))
```

Reply:
I agree, taken into account in 25db0ff